sourcefiles/techdescs.py

```diff
@@ -294,39 +294,68 @@ def update_all_tech_descs(tech_db: techdb.TechDB):
 _single_tech_abbrev = {
```

Comment:
Had to add all the new base tech names here so the tech descriptions could be generated right. We might still update some names, but that seems less important than the functionality, so.